amqp/channel.py

```diff
@@ -219,12 +219,14 @@ def close(self, reply_code=0, reply_text='', method_sig=(0, 0),
             if is_closed:
                 return
 
+            self.is_closing = True
```

Comment:
This property does not seem to be used. Perhaps you considered it as an initial implementation?

Reply:
I decided do the state more generic to cover also channels not only connection (since connection is basically special channel with id 0).